library/src/main/java/com/pokegoapi/api/device/DeviceInfo.java

```diff
@@ -115,6 +119,70 @@ public DeviceInfo(DeviceInfos deviceInfos) {
 	}
 
 	/**
+	 * Gets the default device status for the given api
+	 *
+	 * @param api the api
+	 * @return the default device info for the given api
+	 */
+	public static DeviceInfo getDefault(PokemonGo api) {
+		DeviceInfo deviceInfo = new DeviceInfo();
+		deviceInfo.setDeviceId(api.getUuid());
+		Random random = api.getRandom();
```

Comment:
could be new `Random random = new Random(api.getUuid().hashCode())`; to eliminate the entire Random object in another class, since knowing the UUID may be beneficial but setting your own UUID implies you probably won't want an iPhone based on the UUID.